example.diff

```diff
@@ -1766,3 +1766,78 @@ index 0000000..3b7560e
 +
 +let () =
 +  Alcotest.run "Patch tests" tests
```

Comment:
The existing patch is very long and a lot less interesting than the new one. Perhaps we could keep only the first 4 hunks. The other ones are very large and are about vendored libraries, these should definitely go.